### DISCUSSION TITLE
Model generation error

### DISCUSSION THREAD

**Discussion by Llamalland** [dpapavas/lagrange-keyboard#12] [Q&A] [answered]
I've generated a right.scad using lein and am trying to open it in scad, with no luck. I get:

Loaded design 'C:/AndrewBoxSync/Box Sync/sync/keyboard/lagrange-keyboard-master/things/right.scad'.
[ERROR: Parser error: syntax error in file right.scad, line 14073](https://github.com/dpapavas/lagrange-keyboard/discussions/14073,C:/AndrewBoxSync/Box%20Sync/sync/keyboard/lagrange-keyboard-master/things/right.scad) 
Execution aborted

Same thing happens with left.scad when I generate it. Other scad files are opening just fine, like those from dactyl manuform scad generation. I haven't edited any parameters or files in anyway, it's just the vanilla project at this stage. Any ideas?

**Comment by Llamalland** [comment]
When I auto-replaced "linear_extrude (height=13/12, scale=(80/93 80/93)){" (which seemed to be causing the problem)

with "linear_extrude (height=13/12, scale=80/93){" within openSCAD's editor and saved, that seemed to fix the problem and generate a sensible model, but I would have no idea how to get lein to do the right thing in the first place. But at least I seem to have a workaround of sorts for now!

**Reply by Llamalland** [reply]
And further, when I autoreplaced (80/93 80/93) with [80/93, 80/93], it also rendered perfectly. Maybe it's some kind of deprecated formatting version change problem?

**Comment by Llamalland** [comment, marked as answer]
Completely my bad. I accidentally did the scad-clj-master repo rather than the scale fix. With the scale fix properly installed, everything is great!

**Comment by dpapavas** [comment]
Sorry I couldn't get back to you earlier; I was away during the holidays.  Strange you had that problem with the master branch, since my PR for the issue [has been merged](https://github.com/farrellm/scad-clj/pull/46) long ago.  I suppose the package pulled as a dependency by Leiningen is behind, probably because the version of the `scad-clj` package hasn't been updated since then.
